pipelines/ci-kubernetes-pipeline.yml

```diff
@@ -562,4 +576,78 @@ jobs:
       KUBERNETES_DEPLOYMENT_NAME: pubsubsvc
       KUBERNETES_SELECTOR: app=pubsubsvc
       WAIT_UNTIL_AVAILABLE_TIMEOUT: 200s
-    input_mapping: {docker-image-resource: pubsubsvc-docker-latest}
\ No newline at end of file
+    input_mapping: {docker-image-resource: pubsubsvc-docker-latest}
+
+- name: "Acceptance Tests"
+  serial: true
+  serial_groups: [actionsvc, actionexportersvc, casesvc, collectionexercisesvc, collectioninstrumentsvc, iacsvc, samplesvc, surveysvc, partysvc-stub, pubsubsvc]
+  plan:
+  - get: acceptance-tests-repo
+  - get: acceptance-tests-docker-image
+    trigger: true
+    params:
+      skip_download: true
+  - get: census-rm-kubernetes-handlers-repo
+    trigger: true
+    passed: [actionexportersvc-apply-config]
```

Comment:
Is there a reason `actionexportersvc-apply-config` needs to have passed for this to deploy?

Reply:
Failing the apply config implies the service is either not running up to date or failed to start entirely. We want all the services running against master config before running the tests.